desc/io/input_reader.py

```diff
@@ -1318,7 +1318,7 @@ def parse_vmec_inputs(vmec_fname, threshold=0):  # noqa: C901 - FIXME: simplify
         )
         # scale current profile
         if curr_tor is not None:
-            inputs["current"][:, 1] *= curr_tor / np.sum(inputs["current"][:, 1])
+            inputs["current"][:, 1] *= curr_tor / (np.sum(inputs["current"][:, 1]) or 1)
```

Comment:
what does the " or one" here do?

Reply:
just avoids dividing by zero. IE, if the current profile is all zeros it would end up getting set to NaN, this prevents that case.